postgres/datadog_checks/postgres/config.py

```diff
@@ -58,11 +58,15 @@ def __init__(self, instance):
         self.service_check_tags = self._get_service_check_tags()
         self.custom_metrics = self._get_custom_metrics(instance.get('custom_metrics', []))
         self.max_relations = int(instance.get('max_relations', 300))
+        self.min_collection_interval = instance.get('min_collection_interval', 15)
 
         # Deep Database monitoring adds additional telemetry for statement metrics
         self.deep_database_monitoring = is_affirmative(instance.get('deep_database_monitoring', False))
         # Support a custom view when datadog user has insufficient privilege to see queries
         self.pg_stat_statements_view = instance.get('pg_stat_statements_view', 'pg_stat_statements')
+        # statement samples & execution plans
+        self.pg_stat_activity_view = instance.get('pg_stat_activity_view', 'pg_stat_activity')
+        self.statement_samples_config = instance.get('statement_samples', {}) or {}
```

Comment:
let's document this like: https://github.com/DataDog/integrations-core/blob/b69f5d054d8bd9dae5d35ecc38b8ed6f3d3fccef/mysql/assets/configuration/spec.yaml#L142-L146

Reply:
Oh cool, didn't realize we had this. Added here https://github.com/DataDog/integrations-core/pull/8627/commits/0ea0afb714556fc41f61963a135d7c6698c51280 

I excluded the debug-only setting `run_sync` which doesn't apply for regular check configuration. Only when running manually.